docs/source/quick_start.rst

```diff
@@ -0,0 +1,160 @@
+Quick Start
+============
+
+This guide is intended to jump start new Kokkos users (and beginners, in particular).  Kokkos Core is the foundation for all other Kokkos Ecosystem projects, and is readily usable in your own for performance portability.
+
+
+Download Latest and Build 
+-----------------------------
+
+.. note::
+
+  Please become familiar with `Kokkos Requirements <https://kokkos.org/kokkos-core-wiki/requirements.html>`_, and verify that your machine has all necessary compilers, backend SDK and build system components.
+
+
+..
+ Nota bene:  the link for "Latest" should be stable from one release to the next, but check periodically to be sure 
+..
+
+:bdg-link-primary:`Latest Release <https://github.com/kokkos/kokkos/releases/latest>`
+
+.. code-block:: sh
+  
+  # Uncomment according to the type of file you've downloaded (zip or tar)
+  unzip kokkos-x.y.z.zip 
+  # tar -xzf kokkos-x.y.z.tar.gz
+  cd kokkos-x.y.z
+  mkdir build
+  cd build
+
+
+Basic Configuration
+-------------------
+
+.. note::
+
+  You can create small shell scripts to manage and experiment with configuration details, following the GPU microarchitecture-appropriate examples below.  Upon successful configuration, ``make -j`` to build, and ``make install`` to install.
```

Comment:
Prefer `cmake --build BUILD-DIR` see my previous comments

Reply:
Ping.  We should not suggest to use `make -j`.  As Cedric suggested this is less portable because it assumes that the Unix Makefiles generator is being used.  Also the `-j` option can be problematic (on a login node for instance) because it will use as many processes/jobs as it can detect from where the command is being run.